src/components/table/customisations/SortingLabel.tsx

```diff
@@ -0,0 +1,18 @@
+import { TableHeaderRow } from "@devexpress/dx-react-grid-bootstrap4";
+import { ArrowDropDown, ArrowDropUp } from "@material-ui/icons";
+import React, { FC } from "react";
+
+export const SortingLabel: FC<TableHeaderRow.SortLabelProps> = p => (
+  // there is no nice way to handle sort label clicks and make eslint feel good
+  // eslint-disable-next-line
+  <div
+    onClick={() => p.onSort({ keepOther: false })}
```

Comment:
Could such handlers be given a separate named declaration on top of the file?
This way their purpose would be clearer.
`const sortWithoutOther = props => props.sort({ keepOther: false })`
...
`onClick{sortWithoutOther}`